db-changes/datasets/28102021_alter_study_user.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE study_user RENAME COLUMN receive_anonymization_report TO receive_study_user_report;
```

Comment:
Another thing, this file is not up to date with the new database update from anthony. So when I deploy on qualif, it does not apply automatically

Reply:
moved everything into /docker-compose/database/db-changes